datafusion/src/physical_plan/planner.rs

```diff
@@ -299,12 +299,11 @@ impl PhysicalPlanner for DefaultPhysicalPlanner {
         input_schema: &Schema,
         ctx_state: &ExecutionContextState,
     ) -> Result<Arc<dyn PhysicalExpr>> {
-        DefaultPhysicalPlanner::create_physical_expr(
-            self,
+        create_physical_expr(
```

Comment:
The actual interface for `PhysicalPlanner` remains the same; However it is now also possible to call the free function `create_physical_expr` directly too